vayesta/solver/hamiltonian.py

```diff
@@ -639,7 +640,7 @@ def get_fock(self, with_vext=True, use_seris=True, with_exxdiv=False):
                 + einsum("iipq->pq", gab[oa, oa])  # Coulomb
                 - einsum("ipqi->pq", gbb[ob, :, :, ob])
             )  # Exchange
-            fock = ((fock[0] + dfa), (fock[1] + dfb))
+            fock = np.asarray(((fock[0] + dfa), (fock[1] + dfb)))
```

Comment:
I don't think this can be packed into a single array, since `fock[0]` and `fock[1]` can have different shapes?

Reply:
It can not be if the shapes are different. Wouldn't Fock matrices for alpha and beta always have the same shape?